app/presenters/variant_detail_presenter.rb

```diff
@@ -9,6 +9,7 @@ def as_json(opts = {})
         clinvar_entries: variant.clinvar_entries.map(&:clinvar_id),
         lifecycle_actions: LifecyclePresenter.new(variant),
         sources: variant.sources.map { |s| SourcePresenter.new(s) },
+        allele_registry_id: variant.allele_registry_id,
```

Comment:
If this is going to be on every variant response, it may make sense to add it as a field in the database and lazily populate it as needed rather than fetching it every 24 hrs as we're doing here.

Reply:
Ultimately, we want to display more information from the response (similar to MyVariantInfo) but we wanted this minimal feature out in time for Obi's presentation on Friday. It might still make sense to store this as a field in the database. We'd need to make this a callback to save when the variant gets approved and update should the variant coordinates change.

Reply:
Cool, makes sense! Generally for external sources like this, we don't actually return any of it as part of the civic record (mygeneinfo and myvariantinfo as examples) they just go as a separate request/record, but if we have a good reason to do a hybrid style record in this case (civic variant + external source) this works fine.

Reply:
Yeah, mainly we just wanted to avoid having to make another request to one of our API endpoints on the client side.

Reply:
Yeah, in general I like keeping the stuff that isn't part of the civic record in a different record, but I can see where we may want to try and cut down on requests for efficiency purposes. Perhaps we need a `/variants/:id` for the civic record and then some sort of `/variants/:id/complete` or `/variants/:id/frontend_display` or something that stitches together the disparate records in a single request.

The other consideration is that with one big slow(er) request, we can't render anything until the request is completed. With multiple smaller requests, we can be doing the database work in parallel and render different components as the responses come back, so its not always worse to have multiple requests.